dynamiqs/mesolve/mesolve.py

```diff
@@ -27,118 +27,122 @@ def mesolve(
 ) -> Result:
     r"""Solve the Lindblad master equation.
 
-    Evolve the density matrix $\rho(t)$ from an initial state $\rho(t=0) = \rho_0$
-    according to the Lindblad master equation using a given Hamiltonian $H(t)$ and a
-    list of jump operators $\{L_k\}$. The Lindblad master equation is given by
-
+    This function computes the evolution of the density matrix $\rho_t$, starting
```

Comment:
I don't agree, I've often seen time as a subscript for anything time dependent, which I find more readable (see below). It's also nice when describing discrete schemes, because `\rho_t` becomes `\rho_n` and not `\rho(n)`.

$$
        \frac{\mathrm{d}\rho_t}{\mathrm{d}t} =-i[H_t, \rho_t]
        + \sum_{k=1}^N \left(
            L_k \rho_t L_k^\dagger
            - \frac{1}{2} L_k^\dagger L_k \rho_t
            - \frac{1}{2} \rho_t L_k^\dagger L_k
        \right)
    $$

$$
        \frac{\mathrm{d}\rho(t)}{\mathrm{d}t} =-i[H(t), \rho(t)]
        + \sum_{k=1}^N \left(
            L_k \rho(t) L_k^\dagger
            - \frac{1}{2} L_k^\dagger L_k \rho(t)
            - \frac{1}{2} \rho(t) L_k^\dagger L_k
        \right)
    $$

Also I think we should keep the time dependency when we write Lindblad because of the time-dependent Hamiltonian, we could use the implicit version and just use `H` but it's a bit imprecise.

Reply:
I think it's confusing to have different objects that are time-dependent but not specify the dependency for all the objects.

Reply:
Well I'm not in favor of the subscripted version because I find it quite non-standard. It could be a lack of culture from my end, but I couldn't find papers that use this notation unless they're in the context of SMEs. If you have references I'm fine changing.

For the rest, I don't have super strong opinions. So whether we keep all implicit, half-implicit or fully write time-dependece, it's good with me. Especially if we keep the comment below about time-dependent Hamiltonians.